Add django base 1

kleberbaum opened the pull request:
**Checklist:**

 <!--- Make sure your PR is documented and tested before submission. Put an `x` in all the boxes that apply: -->
 - [ ] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?


**Type of Changes:**

 <!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)


**What is the current behavior? (link to any open issues here)**
- Currently this repository is only an empty shell. #4 
- Currently there is no definition of requirements. #5 


**What is the new behavior (if this is a feature change)?**
- Now it contains the base structure our future Django projects will follow. #4 
- Now there is a definition of requirements. #5

kleberbaum committed:
Add base requirements

I like to split them up into base and production,
because not all python packages work on windows.

kleberbaum committed:
Initialize a new django project

django-admin startproject esite

I typically name my django projects esite as a homage to my old website
https://www.erebos.xyz/ which I define as the root of my current skills.

"Sky's The Limit" - boiz

kleberbaum committed:
Add essential core app

python manage.py startapp core

This app should contain all modifications on the core.

kleberbaum committed:
Add home app as entrypoint for future projects

python manage.py startapp home

This app should always contain the homepage.

Aichnerc commented on esite/core/apps.py:
```diff
@@ -0,0 +1,5 @@
+from django.apps import AppConfig
+
+
+class CoreConfig(AppConfig):
```
Please describe what this part of your code does.

Aichnerc commented on esite/home/apps.py:
```diff
@@ -0,0 +1,5 @@
+from django.apps import AppConfig
+
+
+class HomeConfig(AppConfig):
```
Please describe what this part of your code does.

Aichnerc commented on manage.py:
```diff
@@ -0,0 +1,21 @@
+import sys
+
+
+def main():
```
Please describe what this part of your code does.

kleberbaum committed:
Add comments to apps.py

The quality of the comments has been improved in
esite/core/apps.py and esite/home/apps.py due
to a request of @Aichnerc.
Ref: #6

kleberbaum committed:
Add comments to manage.py

The quality of the comments has been improved in
manage.py due to a request of @Aichnerc.
Ref: #6

kleberbaum reviewed (commented):
all very clean and tidy ^^

Aichnerc commented on esite/home/apps.py:
```diff
@@ -0,0 +1,5 @@
+from django.apps import AppConfig
+
+
+class HomeConfig(AppConfig):
```
Please describe what this part of your code does.

kleberbaum replied:
yikes xD

Aichnerc commented on manage.py:
```diff
@@ -0,0 +1,21 @@
+import sys
+
+
+def main():
```
Please describe what this part of your code does.

kleberbaum replied:
yikes xD

Aichnerc commented on esite/core/apps.py:
```diff
@@ -0,0 +1,5 @@
+from django.apps import AppConfig
+
+
+class CoreConfig(AppConfig):
```
Please describe what this part of your code does.

kleberbaum replied:
yikes xD

Aichnerc reviewed (changes requested):
Please check the comments to see the request changes.

Aichnerc reviewed (approved):
❤🌸💖🌹 Just wonderful. Truly a masterpiece, worthy of honoring in the halls of the eternal queen.

kleberbaum reviewed (commented):
all more clean and tidy ^^
